modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/exportimport/portlet/preferences/processor/DDMFormExportImportPortletPreferencesProcessor.java

```diff
@@ -168,9 +168,20 @@ public PortletPreferences processImportPortletPreferences(
 		long formInstanceId = MapUtil.getLong(
 			formInstanceIds, importedFormInstanceId, importedFormInstanceId);
 
+		long importedGroupId = GetterUtil.getLong(
+			portletPreferences.getValue("groupId", null));
+
+		Map<Long, Long> groupIds =
+			(Map<Long, Long>)portletDataContext.getNewPrimaryKeysMap(
+				Group.class);
+
+		long groupId = MapUtil.getLong(
+			groupIds, importedGroupId, importedGroupId);
```

Comment:
@moltam89, can you please just declare the variable `groupIds` before the variable `importedGroupId`?

I believe that Brian, probably, will ask for it since `groupIds` are used first them `importedGroupId` considering the parameters order on line 179.

If you feel that this is not a big deal, please let me know and I'll move on with the review. :slightly_smiling_face: